thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskRunner.java

```diff
@@ -138,6 +147,19 @@ private void runTask(DateTime windowStart, DateTime windowEnd) throws JobExecuti
     storeData(adOutputContext);
   }
 
+  private double calculateTrafficImpact (MetricTimeSeries metricTraffic, MergedAnomalyResultDTO mergedAnomaly) {
```

Comment:
Is the 'traffic contribution' generic enough to put it here? Does it depend on what time of metric (count, vs average vs other)? If the metric it self is 'count' and it can distributed along one dimension then this TrafficImpact make sense, otherwise its not.